.kitchen.yml

```diff
@@ -1,8 +1,11 @@
 driver:
   name: vagrant
+  customize:
+    cableconnected1: 'on'
 
 provisioner:
   name: chef_zero
+  require_chef_omnibus: 12
 
```

Comment:
We should clean up the port forwards and create a single entry outside of platforms - we're not going to run multiple servers at once so don't require unique ports.